transports/janus_websockets.c

```diff
@@ -755,8 +787,16 @@ int janus_websockets_init(janus_transport_callbacks *callback, const char *confi
 			info.ssl_private_key_password = NULL;
 			info.gid = -1;
 			info.uid = -1;
+			info.options = 0;
+#ifdef __FreeBSD__
+			if (ipv4_only) {
+                            info.options |= LWS_SERVER_OPTION_DISABLE_IPV6;
+                            ipv4_only = 0;
+                        }
```

Comment:
Indentation seems broken? Spaces are used, but we use tabs.

Reply:
fixed